apiserver/facades/schema.json

```diff
@@ -47843,6 +47843,9 @@
         "Description": "UserManagerAPI implements the user manager interface and is the concrete\nimplementation of the api end point.",
         "Version": 3,
         "AvailableTo": [
+            "controller-machine-agent",
+            "machine-agent",
+            "unit-agent",
```

Comment:
Something is panic'ing because we're using something on the context to early. I do have a way to fix this, but the process isn't pretty. For now I think we can ignore this.

Reply:
reverted the changes in `schema.json`